### DISCUSSION TITLE
Unreachable code disassembly

### DISCUSSION THREAD

**Discussion by sgboyd** [NationalSecurityAgency/ghidra#5129] [Q&A]
I know I can disassemble the unreachable code that's formatted as '??' in the GUI by right clicking and selecting disassemble but is there a way to set Ghidra to automatically disassemble all unreachable code without having to individually click on each part of it?

And is there a way to make currentProgram.getListing().getInstructions() include this unreachable code as 'NOP' when scripting?

**Comment by Scotty-Tan** [comment]
I have the exact same question regarding disassembling the unreachable code. Did you manage to find the solution for it?

**Comment by Nemoumbra** [comment]
There's an aggressive instruction finder analyzer, maybe it can help you?
